Cargo.toml

```diff
@@ -20,3 +19,6 @@ members = [
 
 [profile.dev]
 opt-level = 2
+
+[patch.crates-io]
+block-buffer = { git = "https://github.com/RustCrypto/utils", branch = "digest_pad" }
```

Comment:
Note: for whatever reason `cross` seems to have trouble with these directives, so you'll either need to disable the cross-based tests or publish a new release.

There might be other alternatives, like using a `git = "..."` directive in a particular crate's `[dependencies]` section, but I haven't thoroughly investigated.

Reply:
It's not a problem, I plan to publish `pre` version either way. I use `patch` as a way to verify that introduced API changes fit as intended.